webapp/src/components/form/FormTextArea.js

```diff
@@ -59,6 +59,7 @@ class FormTextArea extends React.Component {
             value={this.props.value || ""}
             onChange={this.props.onChange}
             onKeyPress={this.getWordCount}
```

Comment:
Can you remove the onKeyPress handler please, this event was deprecated: see https://developer.mozilla.org/en-US/docs/Web/API/Document/keypress_event